pom.xml

```diff
@@ -2288,6 +2386,9 @@
                     <groupId>org.basepom.maven</groupId>
                     <artifactId>duplicate-finder-maven-plugin</artifactId>
                     <configuration>
+                        <ignoredResourcePatterns combine.children="append">
+                            <ignoredResourcePattern>^git.properties$</ignoredResourcePattern>
```

Comment:
why? if it's conflicting with some other dependency, add this as a exception group, rather then a global ignored resource